webapp/components/experiments/ExperimentEditStyle.js

```diff
@@ -0,0 +1,242 @@
+import {
+  bgDarkest,
+  bgLight,
+  bgRegular,
+  secondaryColor,
+  fontColor,
+  radius,
+  primaryColor,
+  experimentInputColor,
+  experimentFieldColor,
+  experimentSvgColor,
+  experimentLabelColor,
+  experimentAutocompleteBorder,
+  errorFieldBorder,
+} from '../../theme';
+
+/**
+ * Edit/Add view of a single Experiment.
+ *
+ * @return {JSX.Element}
+ * @constructor
+ */
+const useStyles = (theme) => ({
```

Comment:
That's a lot of styling for a single component, we should try to keep it lean as much as possible (not that I want to redo anything, but something to keep in mind for the future)

Reply:
I just extracted existing code to a separate file, I agree with what you're saying